compiler/tests/evm_contracts.rs

```diff
@@ -1238,3 +1281,241 @@ fn self_address() {
         );
     });
 }
+
+#[test]
+fn uniswap_contracts() {
```

Comment:
The file is growing so large that I think we should start to split it up some. I think the Uniswap tests could all go into one dedicated uniswap test file.